tests/scripts/k8s-vagrant-multi-node.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${scriptdir}/../../build/common.sh"
```

Comment:
This script seems in the same spirit as #2736 that it's a nice helper for those that want to use it. It would be good to resolve the question in that PR and then put this in the similar place. 

Also, how about if you add a section in development-environment.md to this as another option for starting a multi-node env?

Reply:
i essentially view this to be the same as `minikube.sh`, a script that can bring up and manage a testing environment.  I think being under `tests/scripts` is OK with me.

I do agree with adding this option to our dev focused documentation, for increased visibility and knowledge about how to use it.